client-service/src/main/docker/wrapper.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+WAITING_FOR_DEPENDENCE=${WAITING_FOR_DEPENDENCE:='false'}
+
+if [ "$WAITING_FOR_DEPENDENCE" != "true" ]; then
+    echo "Starting config server immediately"
+    java -jar ./client-service.jar
```

Comment:
Is this file is autogenerated ? If yes then do we need it to be pushed ? If no then should we always maintain it and support configuration ?